02.extensions/zoofilter/01.basics/03.integration/docs.md

```diff
@@ -77,6 +77,10 @@ If you are using custom elements and those are searchable you would need to assi
 1. Open the custom element XML file and localize the `<element>` tag.
 2. Add a `zfgroup` attribute with the appropriate category value and save. <br /> E.g: `<element type="textpro" zfgroup="input">`
 
+### Not compatible elements with ZOOfilter
+
+The elements that are not compatible with ZOOfilter are: Access, Item Link, State, Author, Edit, Frontpage Enabled, Previous/Next Link, Comments Link, Hits, Print, Searchable, Variations, Measures Pro, ZOOcart, Related Items (Related Items Pro), Related Categories Pro (Category element should be used instead, without support of cross-application in search), Related Categories, Media (MediaPro), Link (LinkPro), Image (ImagePro), GoogleMaps (GoogleMapsPro), Download (DownloadPro), Gallery, E-mail.
```

Comment:
- When both pro and core are in the list reference them Element/Pro, eg "Image/Pro".
- Write them without separation, eg. RelatedItems.
- The comment about Category element should be place below as a note.